.github/actions/todb-tests/entrypoint.sh

```diff
@@ -51,6 +51,23 @@ for file in "$(ls)"; do
 	fi
 done
 
+# Files added must have date and name later than all existing file
+LATEST_FILE="$(ls -t1 |  head -n 1)"
```

Comment:
This should use `git log` to find the latest file.

Reply:
done.